server/server_test.go

```diff
@@ -868,17 +867,13 @@ func (cli *testServerClient) runTestLoadData(t *testing.T, server *Server) {
 		"xxx row4_col1	- 		900\n" +
 		"xxx row5_col1	- 	row5_col3")
 	require.NoError(t, err)
-
-	originalTxnTotalSizeLimit := kv.TxnTotalSizeLimit
-	// If the MemBuffer can't be committed once in each batch, it will return an error like "transaction is too large".
-	kv.TxnTotalSizeLimit = 10240
-	defer func() { kv.TxnTotalSizeLimit = originalTxnTotalSizeLimit }()
-
 	// support ClientLocalFiles capability
 	cli.runTestsOnNewDB(t, func(config *mysql.Config) {
 		config.AllowAllFiles = true
 		config.Params["sql_mode"] = "''"
 	}, "LoadData", func(dbt *testkit.DBTestKit) {
+		// If the MemBuffer can't be committed once in each batch, it will return an error like "transaction is too large".
+		dbt.MustExec("set global tidb_txn_total_size_limit = 10240")
```

Comment:
To be compatible with the code its replacing, it needs to defer a reset to default:
```suggestion
		// If the MemBuffer can't be committed once in each batch, it will return an error like "transaction is too large".
		dbt.MustExec("set global tidb_txn_total_size_limit = 10240")
		defer dbt.MustExec("set global tidb_txn_total_size_limit = default")
```